packages/react-native/Libraries/Image/RCTImageViewManager.mm

```diff
@@ -30,6 +30,7 @@ - (RCTPlatformView *)view // [macOS]
   return [[RCTImageView alloc] initWithBridge:self.bridge];
 }
 
+RCT_REMAP_VIEW_PROPERTY(accessible, reactAccessibilityElementCell.accessibilityElement, BOOL)
```

Comment:
If macOS only, can you put inside a macOS ifdef with tags? 
https://github.com/microsoft/react-native-macos/blob/main/docs/DiffsWithUpstream.md